KarhooSDK/Api/Request/Adyen/AdyenPaymentDetails.swift

```diff
@@ -0,0 +1,83 @@
+//
+//  AdyenPaymentDetails.swift
+//  KarhooSDK
+//
+//  Created by Nurseda Balcioglu on 01/09/2020.
+//  Copyright © 2020 Flit Technologies Ltd. All rights reserved.
+//
+
+import Foundation
+
+public struct AdyenPaymentDetails: Codable, KarhooCodableModel {
+    
+    public let MD: String
+    public let PaReq: String
+    public let PaRes: String
+    public let billingToken: String
+    //public let cupsecureplus.smscode: String
```

Comment:
please remove the commented lines if they are not needed

Reply:
done